bittide/src/Bittide/ClockControl/StabilityChecker.hs

```diff
@@ -1,16 +1,10 @@
 -- SPDX-FileCopyrightText: 2022 Google LLC
 --
 -- SPDX-License-Identifier: Apache-2.0
-{-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE UndecidableInstances #-}
-
 module Bittide.ClockControl.StabilityChecker where
 
 import Clash.Prelude
 
-import Bittide.ClockControl (RelDataCount, targetDataCount)
-import Bittide.ClockControl.Callisto.Util (dataCountToSigned)
-
 -- | Stability results to be returned by the 'stabilityChecker'.
 data StabilityIndication = StabilityIndication
   { stable :: Bool
```

Comment:
Would it maybe make sense to move this to some other module? Now it's the only thing in here.

Reply:
Yeah I'll move it to `Types`